### DISCUSSION TITLE
When is the next release coming?

### DISCUSSION THREAD

**Discussion by dereknola** [vagrant-libvirt/vagrant-libvirt#1448] [General]
It has been almost 3 months since the last release. There have been several PRs that have merged that I am interested in getting access to. When is the next release scheduled to come out?

**Comment by electrofelix** [comment]
Meant to reply, 0.8.0 was cut in April

**Reply by dereknola** [reply]
@electrofelix Thanks so much! I have been watching the releases and was very excited to see v0.8.0 and v0.9.0 come out.
